selfdrive/car/honda/values.py

```diff
@@ -828,6 +829,25 @@ class CAR:
       b'77959-T5R-A230\x00\x00',
     ],
   },
+  CAR.FREED: {
+    # NOTE: omitting gateway ECU since it doesn't appear on all models
```

Comment:
no need to omit, as the gateway isn't an essential ecu

Reply:
I see-- I misunderstood the docs on `match_fw_to_car_exact()`